common/oatbox/log/logger/TaoMonolog.php

```diff
@@ -60,13 +62,23 @@ public function log($level, $message, array $context = array())
         $this->logger->log($level, $message, $context);
     }
 
+    public function getLogger(string $channel = null): LoggerInterface
```

Comment:
Shouldn't class have consistent style -  some new methods are declared in PHP v7.0+ with scalar type declarations, etc, but older - no

Reply:
Agree, want to remind that PHPUnit 4.8 can't handle mocking classes with scalar type hinting